libs/designer-ui/src/lib/panel/recommendationpanel/searchResult.tsx

```diff
@@ -129,8 +137,8 @@ export const OperationActionDataFromOperation = (operation: DiscoveryOperation<D
   id: operation.id,
   title: operation.properties.summary,
   description: operation.properties.description,
-  brandColor: operation.properties.api.brandColor,
-  iconUri: operation.properties.api.iconUri,
+  brandColor: (operation.properties as BuiltInOperation).brandColor || operation.properties.api.brandColor,
```

Comment:
why is this different?

Reply:
Good question. The goal of the feature was to offer operation "aliasing" meaning, although an operation comes from ConnectorA, we'd like for marketing reasons to show it as ConnectorB. This piece of code is used to determine which iconUri and brandColor should be used, so I could use the same logic as above and check displayApi, but I actually saw that an operation itself can declare its own brandcolor and iconUri, so it felt natural to respect it.
Since this is something which was present in previous data model, it was limiting the surface of impact of the displayApi, and I would use displayApi only for grouping, but if you feel I should rather check displayApi here let me know, I'll update.

Reply:
ok last note then, as with he others, please use null coalescing operator rather than the || operator

```suggestion
  brandColor: (operation.properties as BuiltInOperation).brandColor ?? operation.properties.api.brandColor,
```